codice/ddf
# DDF-4270 Remove Workspace Templates

#### What does this PR do?
	- Removes workspace templates from DDF workspace home page
	- Adds `Add New Workspace` button to allow creation of a blank workspace
#### Who is reviewing it? 
@djblue 
@rymach 
@GabrielFabian 
@gjvera 

#### Select relevant component teams: 

#### Ask 2 committers to review/merge the PR and tag them here.
@bdeining
@djblue
#### How should this be tested?
1. Build and install
2. Navigate to `Workspaces` home page
3. Ensure that there is no dropdown in the top right for workspace templates
4. Create a Blank Workspace by clicking the `New Workspace` button
	- Ensure that the workspace does not run a search immediately
5. Mess with theme settings to ensure correct scaling of the `Create Workspace` area
6. Spot test workspace functionality and view logs for any potential regressions

#### Any background context you want to provide?
#### What are the relevant tickets?
[DDF-4270](https://codice.atlassian.net/browse/DDF-4270)
#### Screenshots
Before:
![image](https://user-images.githubusercontent.com/10997562/47527174-1922bd80-d85f-11e8-8b03-cf920c66aa64.png)

After:
![image](https://user-images.githubusercontent.com/10997562/47665701-4fb24e00-db67-11e8-94b5-2bc3532d35d7.png)




#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
